packages/panels/docs/06-navigation.md

```diff
@@ -376,6 +402,8 @@ public function panel(Panel $panel): Panel
 }
 ```
 
+<AutoScreenshot name="panels/navigation/custom-items" alt="Register Custom Navigation Items" version="3.x" />
```

Comment:
Please do not use Title Case for the alt text within screenshots, please check this is consistent with existing screenshots :)